diffusers/training_example.ipynb

```diff
@@ -0,0 +1,3955 @@
+{
```

Comment:
<p>I'm not a wand expert, but maybe have the user fill in this information / use something neutral  <code>entity': 'tmabraham',</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I added a note to change it, but also it will ask for authentication...